src/crashlog.h

```diff
@@ -114,6 +114,12 @@ class CrashLog {
 	 */
 	static void InitialiseCrashLog();
 
+	/**
+	 * Initialiser in case a newly started thread has to be prepared for the crash log.
```

Comment:
this is a weird sentence .. initialiser in case a thread has to be prepared .. feels really vague :D Can't you make it more absolute in what it does / when it has to be called?

Reply:
I just copied the language from the comment for `InitialiseCrashLog` directly above, but you are right that it reads a bit strange.